### PR TITLE
Add to parcel name replacements

### PR DESCRIPTION
**Pull request by mgraber**
#121

### CODE REVIEW

File: colp_build/sql/name.sql [hunk 1]
```diff
@@ -9,104 +9,110 @@ SELECT
...
+																						REGEXP_REPLACE(
+																							REGEXP_REPLACE(
+																								REGEXP_REPLACE(
+																									REGEXP_REPLACE(
+																										REGEXP_REPLACE(
+																											REGEXP_REPLACE(' '||TRIM(parcel_name)||' ', 
+																											'\s&\s|&', 
+																											' AND '
+																											), 
+																										'PLAYLOT\s|PLGD\s|PLAYGRD\s|PLGRD\s|PLAYGND\s|PLYGRND/|PLYGRND\s|PLYGD\s|PLYGD|PLAYGRND\s|PLYGRD\s|PLYGRD/|PLGRD/|PLGRD\s|PLGD.\s|PLGD\s|PLYGD/|PLYGD|PLYGD\s|PLAYGD\s|PLGND\s|PLGD/|PLGD-|PLAYGND-',
```
Comment (AmandaDoyle) [inline colp_build/sql/name.sql:40]: PLYGRND is repeated

Comment (AmandaDoyle) [inline colp_build/sql/name.sql:40]: Note: it'd be useful to have you explain why some phrases that are getting replaced have white space characters or not.  Fo example, you have `PLGRD/|PLGRD\s` but just `PLAYLOT\s`

File: colp_build/sql/name.sql [hunk 2]
```diff
@@ -9,104 +9,110 @@ SELECT
...
+																									'\@',
+																									'AT'
+																									),
+																								'HSES',
+																								'HOUSES'
+																								),
+																							'\s\(GROUP 1\s',
+																							'\s(GROUP 1)\s'
+																							),
+																						'\sPK\s|\sPK-',
```
Comment (AmandaDoyle) [inline colp_build/sql/name.sql:52]: Note: Is there any risk of this accidently replacing Pre-K?  Check in QAQC review.

**Review by AmandaDoyle — APPROVED** [review]

File: colp_build/sql/name.sql (hunk 1 above)
Comment (mgraber) [reply colp_build/sql/name.sql:40]: Replacements are happening to `' '||TRIM(parcel_name)||' '`. The spaces get added to more easily identify ends of words. Examples where `PLGRD/` would get replaced include: `PS 192 PLGRD/JACOB SCHIFF` and `P.S.28 PLGRD/HIGHBRIDGE PARK`. Examples where `PLGRD\s` would get replaced include: `JHS 144 & L B BURNS PLGRD` and `S ST. CPORT/BURLING SLIP/PLGRD`

**Pull request merged**